pkgs/development/libraries/avro-c++/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchurl, cmake, boost155, python
```

Comment:
If you're only interested in the python interpreter, I believe the preferred method of getting at it is passing in `pythonPackages` and using `pythonPackages.python` in the build inputs.